.github/workflow/ktlint-action.yml

```
@@ -0,0 +1,25 @@
+        with:
+          fetch-depth: 1
+
+      - name: Setup JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: '17'
+          distribution: 'adopt'
+
+      - name: Run ktlint
+        run: |
+          ./gradlew ktlintCheck --daemon --parallel --configure-on-demand
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)